### PR TITLE
Bugfix: ScrollBar Issue

### PR DESCRIPTION
**Pull request by sebastian-dev**
Scrollbars not working with Mouse Drag
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/1406

### CODE REVIEW

**Comment by rmattuschka** [comment]
very good thank you!

File: LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs [hunk 1]
```diff
@@ -39,6 +42,57 @@ public ThemedHScrollIndicator(HScrollBar scrollBar)
             scrollBar.ValueChanged += (s, e) => Invalidate();
 
             scrollBar.Height = 0;
+            this.MouseDown += OnMouseDown;
+        }
+
+        private void OnMouseDown(object sender, MouseEventArgs e)
+        {
+            if (_isScrolling)
```
Comment (PhyxionNL) [inline LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs:50]: Can you reformat both changed files please? Thanks :)

Comment (sebastian-dev) [reply LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs:50]: reformat depend on the editorconfig rules.

File: LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs [hunk 2]
```diff
@@ -39,6 +42,57 @@ public ThemedHScrollIndicator(HScrollBar scrollBar)
...
+                return;
+            _isScrolling = true;
```
Comment (PhyxionNL) [inline LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs:52]: This needs a new line between return statement (also in other doc).

File: LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs [hunk 3]
```diff
@@ -39,6 +42,57 @@ public ThemedHScrollIndicator(HScrollBar scrollBar)
...
+        private void OnMouseMove(object sender, MouseEventArgs e)
+        {
+            if (!_isScrolling)
+                return;
+
+            //pixel to range scaling:
+            double totalRange = _scrollbar.Maximum - _scrollbar.Minimum;
+
+            if (totalRange <= 0)
+            {
```
Comment (PhyxionNL) [inline LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs:79]: Single line blocks don't need { }, same below and in other doc.

File: LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs [hunk 4]
```diff
@@ -39,6 +42,57 @@ public ThemedHScrollIndicator(HScrollBar scrollBar)
...
+
+            if (scrollValue < _scrollbar.Minimum)
+            {
+                scrollValue = _scrollbar.Minimum;
+            }
+            if (scrollValue > (_scrollbar.Maximum - _scrollbar.LargeChange))
+            {
+                scrollValue = _scrollbar.Maximum - _scrollbar.LargeChange;
+            }
+            _scrollbar.Value = (int)scrollValue;
```
Comment (PhyxionNL) [inline LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs:94]: Insert new line before 🙂

**Review by PhyxionNL — CHANGES_REQUESTED** [review]

File: LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs (hunk 4 above)
Comment (sebastian-dev) [reply LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs:91]: the requested changes are online

**Pull request merged**